java/code/src/com/redhat/rhn/domain/action/script/ScriptRunAction.java

```diff
@@ -64,7 +64,7 @@ public String getHistoryDetails(Server server, User currentUser) {
                         .getReturnCode().toString()));
                 retval.append("</br>");
                 retval.append(ls.getMessage("system.event.scriptRawOutput"));
-                retval.append("<a href=\"" +
+                retval.append("<a target=\"_blank\" href=\"" +
```

Comment:
While I see this could work, and it could even be better in the end to open it into a different tab, I believe the bug is due to the single page application behavior. To properly fix the bug and make it work as it was before, we should disable the `spa` behavior with the `data-senna-off` attribute. But I personally even like better to open it into a different tab: in the end we are no longer into the product webUI, but we are displaying a **raw output** into the browser.
So, in the end, we should add them both the attributes: one to make sure the application does not try to behave as a `spa`, the other one to open it into a new tab.

Reply:
Makes sense!
I updated the PR with you suggestion please have another look and approve if you're satisfied :smile: